trie/syncTrieStorageManager_test.go

```diff
@@ -0,0 +1,63 @@
+package trie
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/testscommon/trie"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSyncTrieStorageManagerInvalidStorerType(t *testing.T) {
+	_, trieStorage := newEmptyTrie()
+
+	stsm, err := NewSyncTrieStorageManager(trieStorage)
+	assert.Nil(t, stsm)
+	assert.True(t, strings.Contains(err.Error(), "invalid storer type"))
+}
+
+func TestNewSyncTrieStorageManager(t *testing.T) {
+	_, trieStorage := newEmptyTrie()
+	trieStorage.mainStorer = &trie.SnapshotPruningStorerStub{}
+	stsm, err := NewSyncTrieStorageManager(trieStorage)
+	assert.Nil(t, err)
+	assert.NotNil(t, stsm)
+}
+
+func TestNewSyncTrieStorageManager_PutInFirstEpoch(t *testing.T) {
```

Comment:
please change this test. This does not properly test that we do only 1 put. (change putInEpochCalled from bool to int)

Reply:
Done.